pkg/instrumentation/python.go

```diff
@@ -71,6 +72,15 @@ func injectPythonSDK(pythonSpec v1alpha1.Python, pod corev1.Pod, index int) (cor
 		})
 	}
 
+	// Set OTEL_EXPORTER_OTLP_PROTOCOL to http/protobuf if not set by user because it is what our autoinstrumentation supports.
+	idx = getIndexOfEnv(container.Env, envOtelExporterOTLPProtocol)
```

Comment:
If we set the top-level env var we no longer need to set the signal specific-env vars.

Reply:
Yep, will drop signal specific env vars

Reply:
Updated PR